tests/macsec/test_macsec_mka_traffic.py

```diff
@@ -135,6 +135,20 @@ def test_encrypt_with_mka(api, b2b_raw_config, utils):
     # Flow
     f1 = config.flows.flow(name="f1")[-1]
 
+    # IP
```

Comment:
We need to validate the RAW scenario when packets has `macsec` e,g, `f1.packet.ethernet().macsec().ipv4()` . 
```
    flow.tx_rx.port.tx_name = tx.name
    flow.tx_rx.port.rx_name = rx.name
```
Then it will call `_append_header` to configure `macsec`. We need to check we have missed something in that path.